docs/proposals/20220825-azuremanaged-cluster-v1.md

```diff
@@ -0,0 +1,67 @@
+---
+title: AzureManagedCluster graduation to v1 from experimental
+authors:
+  - "@jackfrancis"
+reviewers:
+  - @CecileRobertMichon
+  - @zmalik
+  - @NovemberZulu
+creation-date: 2022-08-25
+last-updated: 2022-08-25
+status: implementable
+see-also:
+- https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2204
+- https://github.com/kubernetes-sigs/cluster-api/pull/6988
+---
+
+
+# AzureManagedCluster graduation to v1 from experimental
```

Comment:
what do we mean by "v1" exactly? Are we talking about the API version (currently already v1beta, I'm not sure we want to graduate the API the v1 quite yet, as the rest of Cluster API CRDs are on v1beta1 still), or are we talking about removing the feature flag and moving the API to `api/` in the codebase instead of `exp/api`? I'm assuming it's the latter but would be good to be explicit, "v1" doesn't mean the same thing to everyone

Reply:
I've replaced references to "v1" with "graduation from experimental". I agree that the term "v1" doesn't have much concrete meaning, and defining that (IMO) isn't part of this scope.